.github/buildomat/jobs/tuf-repo.sh

```diff
@@ -8,6 +8,9 @@
 #:	"=/work/manifest.toml",
 #:	"=/work/repo.zip",
 #:	"=/work/repo.zip.sha256.txt",
+#:	"=/work/helios.json",
+#:	"=/work/incorporation.p5m",
+#:	"=/work/incorporation.p5p",
```

Comment:
@jclulow this won't throw an error if these files aren't present on release branches, right? Or do I need a different sigil here to make these "optional"?